project/Makefile.twig

```diff
@@ -20,3 +20,9 @@ test:
 
 docs:
 	cd {{ docs_path }} && sphinx-build -W -b html -d _build/doctrees . _build/html
+{% if assets_path %}
+
+assets:
```

Comment:
Would be great to do something like this, or even move to yarn and do exactly this: https://stackoverflow.com/questions/44036997/how-to-prevent-yarn-install-from-running-twice-in-makefile#44226605

Reply:
Yarn is not a build system. It's a tool for dependency management.

Reply:
Now that I think of it, having a `.lock` in a library is probably not a good idea anyway

Reply:
But I don't know how npm and yarn differ so much that it's ok to use one and not the other.

Reply:
Changed `npm` to `yarn`, because this is a lot faster. A `.lock` file is optional (same as composer.lock). So this is okay